### ISSUE TITLE
Info box: show the number too, not only the label

### ISSUE THREAD

**Issue by justanotheranonymoususer** [x64dbg/x64dbg#1588]
Screenshot:

![here](http://i.imgur.com/Jmwx7m3.png)

**Comment by blaquee** [comment]
Is that line related to the currently selected line?

**Comment by mrexodia** [comment]
Yes

**Comment by blaquee** [comment]
@justanotheranonymoususer see if the recent PR accomplished your request. then close if so, thanks.

**Comment by justanotheranonymoususer** [comment]
Yes, it does, thanks! I'd say that it would be also useful to show addresses here, near the menu entries:

![image](https://user-images.githubusercontent.com/5781692/31305944-09342db2-ab4e-11e7-937e-a8d4e686e011.png)

And that's nitpicking, but I'd do the formatting the other way around, i.e.: `rax=12345678 (label)`

**Comment by mrexodia** [comment]
Agreed with @justanotheranonymoususer, it should be similar to the memory address:

![](https://i.imgur.com/g0dOOjv.png)

**Comment by blaquee** [comment]
Ok. np
